### PR TITLE
added project_name field

### PR DESCRIPTION
**Pull request by krunalhinguu**
##### ISSUE TYPE

 - Bug fix Pull Request

##### SUMMARY

project_name field was made mandatory in new goreleaser version, due to this make container-build was failing
added this field in .gorelease.yml to fix this issue

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -1,58 +1,59 @@
...
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+    goarch:
+      - amd64
+      - arm
+      - arm64
+    goarm:
+      - 7
```
Comment (PrasadG193) [inline .goreleaser.yml:23]: @krunalhinguu is this change necessary?

Comment (krunalhinguu) [reply .goreleaser.yml:23]: should i delete??

Comment (krunalhinguu) [reply .goreleaser.yml:23]: i thought you told me to raise the PR

Comment (krunalhinguu) [reply .goreleaser.yml:23]: or you talking about autofomating?? @PrasadG193

Comment (PrasadG193) [reply .goreleaser.yml:23]: Yes, I was talking about the formatting changes. Are those intentional? Does exiting format causing any issues?

**Review by PrasadG193 — APPROVED** [review]
